gp-includes/things/translation-set.php

```diff
@@ -260,6 +260,11 @@ public function import( $translations, $desired_status = 'current' ) {
 			 */
 			$entry->status = apply_filters( 'gp_translation_set_import_status', $is_fuzzy ? 'fuzzy' : $desired_status );
 
+			$entry->warnings = maybe_unserialize( GP::$translation_warnings->check( $entry->singular, $entry->plural, $entry->translations, $locale ) );
+			if ( ! empty( $entry->warnings ) ) {
+				$entry->status = 'waiting';
```

Comment:
Should we move this even farther up to above the filter so that the filter could override the warning status?